src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,4 @@
+*REMOVED*abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.TypeExpression(Microsoft.CodeAnalysis.ITypeSymbol typeSymbol) -> Microsoft.CodeAnalysis.SyntaxNode
```

Comment:
This is the change of an abstract method to a non-virtual one.  This is binary safe to make as this type is not publicly sublcassable (it has an internal constructor).